src/rule.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+import {Document} from 'polymer-analyzer/lib/model/document';
+import {Warning} from 'polymer-analyzer/lib/warning/warning';
+
+// TODO(usergenic): Linter rules should have a standard interface for
+// configuring Warning severity and for setting their options.
```

Comment:
I suspect we'll want to do warning severity filtering and modification in a central place, downstream of individual rules